source/upgrade.txt

```diff
@@ -152,6 +152,20 @@ Version 5.0 Breaking Changes
   ``BsonDecimal128.isNumber()`` now returns ``true``, and
   ``BsonDecimal128.asNumber()`` returns the equivalent ``BsonNumber``.
 
+- This driver version removes the following classes from the
+  ``com.mongodb.event`` package:
+  
+  - ``ClusterListenerAdapter``
+  - ``ConnectionAddedEvent``
+  - ``ConnectionPoolListenerAdapter``
+  - ``ConnectionPoolOpenedEvent``
+  - ``ConnectionRemovedEvent``
+  - ``ServerListenerAdapter``
+  - ``ServerMonitorListenerAdapter``
```

Comment:
`ConnectionAddedEvent`, `ConnectionPoolOpenedEvent`, `ConnectionRemovedEvent` are events, so the corresponding methods of `ConnectionPoolListener` were also removed: `connectionAdded`, `connectionPoolOpened`, `connectionRemoved` (unfortunately, they are not specified in the description of DOCSP-34456). Let's mention that these methods were removed.